src/libdredd/src/mutation_replace_expr.cc

```diff
@@ -77,12 +88,25 @@ std::string MutationReplaceExpr::GetFunctionName(
 bool MutationReplaceExpr::ExprIsEquivalentTo(const clang::Expr& expr,
                                              int constant,
                                              clang::ASTContext& ast_context) {
```

Comment:
I am not sure I like the use of `int` for all three data types here.

I think it would be better to have three separate methods, `ExprIsEquivalentToInt`, `ExprIsEquivalentToFloat` and `ExprIsEquivalentToBool`, taking `int`, `double` and `bool` arguments respectively.

Reply:
Refactored.